docs/level1/02-draw-all-elements-from-response.md

```diff
@@ -1,6 +1,7 @@
 ## Step 2
 
-Replace the last `then` callback content to print out all the items in the database using `forEach`
+Replace the content of the last `then` callback to print out all the items we have.
+We do this with the the help of the `forEach` funciton
```

Comment:
I believe is better to use `method` instead of `function`.

ps: there's a typo : )